tests/vmi_lifecycle_test.go

```diff
@@ -485,6 +485,37 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			})
 		})
 
+		Context("when guest crashes", Label("crashing-guest"), decorators.VMIlifecycle, func() {
+			It("should be stopped and have Failed phase when a PanicDevice is provided", func() {
+				vmi := libvmifact.NewFedora(libvmi.WithPanicDevice(v1.Isa))
+				vmi, err := kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred(), "Should create VMI successfully")
+				libwait.WaitUntilVMIReady(vmi, console.LoginToFedora)
+
+				ctx, cancel := context.WithCancel(context.Background())
+				defer cancel()
+
+				By("Crashing the vm guest")
+				Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
+					&expect.BSnd{S: "sudo su -\n"},
+					&expect.BExp{R: "#"},
+					&expect.BSnd{S: `echo c > /proc/sysrq-trigger` + "\n"},
+					&expect.BExp{R: "#"},
```

Comment:
Wouldn't this flake in case the guest crashes before a new line is printed?

Reply:
The test was failing if I didn't expect a new line. I'm suspecting the console wasn't sending the command without L503.

Reply:
```
  ESC[34mSent:ESC[39m "echo c > /proc/sysrq-trigger\n"
  ESC[32mMatch for RE:ESC[39m "echo c > /proc/sysrq-trigger((?s).*)sysrq triggered crash" found: ["echo c > /proc/sysrq-trigger\r\n[   11.766720] sysrq: Trigger a crash\r\n[   11.768184] Kernel panic - not syncing: sysrq triggered crash" "\r\n[   11.766720] sysrq: Trigger a crash\r\n[   11.768184] Kernel panic - not syncing: "] Buffer:  echo c > /proc/sysrq-trigger
  [   11.766720] sysrq: Trigger a crash
  [   11.768184] Kernel panic - not syncing: sysrq triggered crash
```
That `#` was incorrect to look for. I twas just working out because the character was eventually getting printed out in `dmesg` after the crash. 

I've changed it to wait for `sysrq triggered crash` instead. This should help avoid flakes